docs/topics/images.rst

```diff
@@ -1,76 +1,94 @@
 .. _topics-images:
 
-=======================
-Downloading Item Images
-=======================
+===========================================
+Downloading and processing files and images
+===========================================
 
 .. currentmodule:: scrapy.contrib.pipeline.images
 
-Scrapy provides an :doc:`item pipeline </topics/item-pipeline>` for downloading
-images attached to a particular item, for example, when you scrape products and
-also want to download their images locally.
+Scrapy provides reusable :doc:`item pipelines </topics/item-pipeline>` for
+downloading fies attached to a particular item (for example, when you scrape
+products and also want to download their images locally). These pipelines share
+a bit of functionality and structure (we refer to them as media pipelines), but
+typically you'll either use the Files Pipeline or the Images Pipeline.
 
-This pipeline, called the Images Pipeline and implemented in the
-:class:`ImagesPipeline` class, provides a convenient way for
-downloading and storing images locally with some additional features:
+Both pipelines implement these features:
+
+* Avoid re-downloading media that was downloaded recently
+* Specifying where to store the files (filesystem directory, Amazon S3 bucket)
```

Comment:
I think we should refer to _media_ and not _files_ or _images_ when talking in generics.